kerby-kerb/kerb-admin-server/src/main/java/org/apache/kerby/kerberos/kerb/admin/server/kadmin/AdminServerHandler.java

```diff
@@ -103,6 +104,10 @@ public ByteBuffer handleMessage(ByteBuffer receivedMessage,
                 System.out.println("message type: export keytab req");
                 responseMessage = handleExportKeytabReq(localKadmin, fieldInfos);
                 break;
+            case CHANGE_PWD_REQ:
+                System.out.println("message type changePwd req");
```

Comment:
This message is better to be "message type: change password req".

Reply:
Already fixed.